:func:`pytest.skip(msg=...) <pytest.skip>` and :func:`pytest.fail(msg=...) <pytest.fail>` signatures now accept a ``reason`` argument instead
of ``msg``.  Using ``msg`` still works, but is deprecated and will be removed in a future release.

This was changed for consistency with :func:`pytest.mark.skip <pytest.mark.skip>` and  :func:`pytest.mark.xfail <pytest.mark.xfail>` which both accept
``reason`` as an argument.
